app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Helper methods for session handling
+module SessionsHelper
+  def log_in(token)
```

Comment:
This should be in the Application controller.

Reply:
It saves a token in the client side in a cookie. no reason to put it in the controller.

Reply:
But you are putting it in the controller, this is what I'm saying, you have included the helper in the Controller because it is needed by the controllers.